packages/date-time/src/components/Calendar/Calendar.types.ts

```diff
@@ -3,6 +3,7 @@ import { IRefObject, IBaseProps, IStyleFunctionOrObject } from '@uifabric/utilit
 import { IStyle, ITheme } from '@uifabric/styling';
 import { ICalendarDayProps, ICalendarDayGridStyles } from './CalendarDay/CalendarDay.types';
 import { ICalendarMonthProps } from './CalendarMonth/CalendarMonth.types';
+import { IGridStrings } from '@fluentui/date-time-utilities/lib/dateGrid/DateGrid.types';
```

Comment:
Please use package imports rather than path based imports.

Reply:
Oh, sure :)
I thought that /lib/ imports are preferred way, will update those :)